test/setup-wp-aliases.js

```diff
@@ -5,6 +5,7 @@
 	'components',
 	'utils',
 	'blocks',
+	'blocksapi',
```

Comment:
I am not sure what this should be but `blocksapi` isn't used anywhere else in this PR